kubernetes/onpremise/kubeadm/scripts/install_kubernetes.sh

```diff
@@ -0,0 +1,177 @@
+# install Kubectl
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+curl -LO "https://dl.k8s.io/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl.sha256"
```

Comment:
Please get the version number in a separate command:
```suggestion
VERSION="$(curl -fsSL https://dl.k8s.io/release/stable.txt)"
curl -fsSLO "https://dl.k8s.io/release/$VERSION/bin/linux/amd64/kubectl"
curl -fsSLO "https://dl.k8s.io/$VERSION/bin/linux/amd64/kubectl.sha256"
```